channels/us.m3u

```diff
@@ -6373,9 +6373,9 @@ https://ad-playlistserver.aws.syncbak.com/playlist/13613390/master.m3u8?access_t
 https://lnc-waga-fox-aws.tubi.video/index.m3u8
 #EXTINF:-1 tvg-id="WarnerTV.us" tvg-name="Warner TV" tvg-country="US" tvg-language="English" tvg-logo="http://static.epg.best/fr/WarnerTV.fr.png" group-title="Movies",Warner TV
 http://203.154.243.89:11205/
-#EXTINF:-1 tvg-id="WarnerTV.us" tvg-name="Warner TV" tvg-country="US" tvg-language="English" tvg-logo="http://static.epg.best/fr/WarnerTV.fr.png" group-title="",Warner TV
+#EXTINF:-1 tvg-id="WarnerTV.us" tvg-name="Warner TV" tvg-country="US" tvg-language="English" tvg-logo="http://static.epg.best/fr/WarnerTV.fr.png" group-title="Movies",Warner TV
 https://liveanevia.mncnow.id/live/eds/WarnerTV/sa_dash_vmx/WarnerTV.mpd
-#EXTINF:-1 tvg-id="WarnerTV.us" tvg-name="WarnerTV" tvg-country="TH" tvg-language="Thai" tvg-logo="https://www.img09.xyz/assets/img/ch_logo/hd-warnertv.png" group-title="Movies",WarnerTV (720p) [Not 24/7]
+#EXTINF:-1 tvg-id="WarnerTV.us" tvg-name="WarnerTV" tvg-country="TH" tvg-language="Thai" tvg-logo="http://static.epg.best/fr/WarnerTV.fr.png" group-title="Movies",WarnerTV (720p) [Not 24/7]
```

Comment:
Name inconsistency. Either it's Warner TV, either it's WarnerTV.

Reply:
Good catch